src/client/js/components/main.jsx

```diff
@@ -51,7 +51,7 @@ module.exports = React.createClass({
         {this.isThereAGameToday(nextGame)}
 
         <h2>{ this.props.homeTeam } vs. the fucking { nextGame.opponent }</h2>
-        <h3>{ moment(nextGame.date).format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
+        <h3>{ moment(nextGame.time, 'h:ma Z').format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
```

Comment:
That's rough. If the MLB source data is wrong, maybe the data file should be fixed manually until the MLB fixes it.

The parsing on this line is not be including the timezone and must be incorrect outside of MST.